lib/cli/src/commands/package/unpack.rs

```diff
@@ -0,0 +1,90 @@
+use std::path::PathBuf;
+
+use anyhow::Context;
+
+/// Extract contents of a container to a directory.
+#[derive(clap::Parser, Debug)]
+pub struct CmdPackageUnpack {
+    /// The output directory.
+    #[clap(short = 'o', long)]
+    out_dir: PathBuf,
+
+    /// Overwrite existing directories/files.
+    #[clap(long)]
+    overwrite: bool,
+
+    /// Path to the package.
+    package_path: PathBuf,
+}
+
+impl CmdPackageUnpack {
+    pub(crate) fn execute(&self) -> Result<(), anyhow::Error> {
+        eprintln!("Unpacking...");
```

Comment:
We should be able to use `indicatif` here. It'll give us nice things like a spinner and automatically hide output if stderr isn't a TTY.

Reply:
Let's do this in a followup.